### PR TITLE
Merge 18.5.1 changes into `trunk`

### PR DESCRIPTION
**Pull request by mokagio**
Leaving this as a draft till we actually release it. I'd appreciate a review anyway.

Thanks 🙇‍♂️ .

- [x] WordPress submitted for App Store Review
- [x] Jetpack submitted for App Store Review
- [x] WordPress approved by Apple and started phased release
- [x] Jetpack approved by Apple and started phased release

### CODE REVIEW

**Review by AliSoftware — APPROVED** [review]
✅  Diff from the hotfix is cherry-picked this PR's diff
✅  This PR to trunk includes version bumps to the hotfix version number

Ready to :shipit: – but I took the liberty to disable auto-merge because I noticed that you still have 2 items unchecked in the PR descriptions' checklist, and I assume that like me you prefer to wait for Apple to approve the hotfix for WP and JP and them to start rolling the updates out before actually merging the PR.

**Comment by mokagio** [comment]
> but I took the liberty to disable auto-merge [...]

Good one. Thanks! I mean to enable it to get ahead, but I didn't realize I had already made the PR "Ready for review".

Actually... Had I not marked is as "Ready for review" I wouldn't have been able to turn on auto-merge... _That was a brain fart on my side._

**Comment by mokagio** [comment]
18.5.1 released. Merging...

**Pull request merged**
